auslib/web/admin/swagger/api.yaml

```diff
@@ -36,129 +50,418 @@ tags:
       url: "http://mozilla-balrog.readthedocs.io/en/latest/database.html#users"
 
   - name: "CSRF Token"
-    description: "CSRF Token Description"
+    description: "Cross-site request forgery Token for preventing execution of unauthorized commands"
     externalDocs:
       description: "Find out more about CSRF-Token"
       url: "http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#csrf-token"
 
-# basePath: "/api"
-# host: "localhost:8080"
-# Swagger-UI bug : fix TODO week 13
+  - name: "Releases"
+    description: "Balrog Releases Description"
+    externalDocs:
+      description: "Find out more about Releases"
+      url: "http://mozilla-balrog.readthedocs.io/en/latest/database.html#releases"
+
+parameters:
```

Comment:
So if "product" ends up getting used across multiple endpoints it should be moved to this section?